test/book/specialized/fang-fourier-monomial-bases/sl7-cases.jlcon

```diff
@@ -0,0 +1,17 @@
+[fundamentals] => 320
+[fundamentals, [1, 0, 1, 0, 0]] => 70
+[fundamentals, [0, 0, 1, 0, 1]] => 70
+[fundamentals, [0, 1, 0, 1, 0]] => 54
+[fundamentals, [1, 0, 0, 1, 0]] => 28
+[fundamentals, [0, 1, 0, 0, 1]] => 28
+[fundamentals, [1, 0, 0, 1, 0], [0, 1, 0, 1, 0]] => 46
+[fundamentals, [0, 1, 0, 1, 0], [0, 1, 0, 0, 1]] => 46
+[fundamentals, [1, 0, 1, 0, 0], [1, 0, 0, 1, 0]] => 30
+[fundamentals, [0, 1, 0, 0, 1], [0, 0, 1, 0, 1]] => 30
+[fundamentals, [1, 0, 1, 0, 0], [0, 1, 0, 0, 1]] => 8
+[fundamentals, [1, 0, 0, 1, 0], [0, 0, 1, 0, 1]] => 8
+[fundamentals, [1, 0, 1, 0, 0], [1, 0, 0, 1, 0], [0, 1, 0, 1, 0]] => 70
+[fundamentals, [0, 1, 0, 1, 0], [0, 1, 0, 0, 1], [0, 0, 1, 0, 1]] => 70
+[fundamentals, [1, 0, 0, 1, 0], [0, 1, 0, 1, 0], [0, 1, 0, 0, 1]] => 10
+[fundamentals, [1, 0, 1, 0, 0], [1, 0, 0, 1, 0], [0, 1, 0, 1, 0], [0, 1, 0, 0, 1]] => 10
+[fundamentals, [1, 0, 0, 1, 0], [0, 1, 0, 1, 0], [0, 1, 0, 0, 1], [0, 0, 1, 0, 1]] => 10
```

Comment:
this is no code, but just some output formatted as code. This file was already in the first round of the book but was not imported here back then.